src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,35 +1,36 @@
 package edu.kis.vh.nursery;
 
-import edu.kis.vh.nursery.list.IntLinkedList;
+import edu.kis.vh.nursery.stacks.IntLinkedList;
+import edu.kis.vh.nursery.stacks.IIntStack;
 
 public class DefaultCountingOutRhymer {
-private IntLinkedList intLinkedList;
+private IIntStack iIntStack;

```

Comment:
attribute name "i" prefix does not hold any information concerning its meaning
formatting incorrect?